src/utils.py

```diff
@@ -191,6 +191,40 @@ def set_test_env(images, use_latest_additional_tag=False, images_env="DLC_IMAGES
     write_to_json_file(constants.TEST_ENV_PATH, test_envs)
 
 
+def get_safety_scan_allowlist_path(image_uri):
+    """
+    Retrieves the safety_scan_allowlist_path for each image_uri.
+
+    :param image_uri: str, consists of f"{image_repo}:{image_tag}"
+    :return: string, safety scan allowlist path for the image
+    """
+    from test.test_utils import get_ecr_scan_allowlist_path
+
+    os_scan_allowlist_path = get_ecr_scan_allowlist_path(image_uri)
+    safety_scan_allowlist_path = os_scan_allowlist_path.replace(".os_", ".py_")
+    return safety_scan_allowlist_path
+
+
+def get_safety_ignore_dict_from_image_specific_safety_allowlists(image_uri):
+    """
```

Comment:
Add a comment in the places where this function is used and explain what is being done at that point

Reply:
Add detail on what is being combined with this at every usage location

Reply:
Done